nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/dom/Http.java

```diff
@@ -69,7 +69,7 @@ public interface Http extends ConfigBeanProxy, PropertyBag {
     boolean DNS_LOOKUP_ENABLED = false;
     boolean RCM_SUPPORT_ENABLED = false;
     boolean TIMEOUT_ENABLED = true;
-    boolean TRACE_ENABLED = false;
+    boolean TRACE_ENABLED = true;
```

Comment:
Not keen on this, explanation in Jira.

Reply:
this was reverted, this is not ideal solution